packages/react/fast-components-styles-msft/src/accent-button/index.ts

```diff
@@ -24,14 +24,16 @@ import {
     highContrastSelectedOutline,
     highContrastSelector,
 } from "../utilities/high-contrast";
+import { accentFillGradientRest } from "../utilities/color/accent-fill-gradient";
 
 const styles: ComponentStyles<AccentButtonClassNameContract, DesignSystem> = {
     ...baseButton,
     button: {
         ...buttonStyles(),
         color: accentForegroundCut,
         fill: accentForegroundCut,
-        background: accentFillRest,
+        // background: accentFillRest,
+        background: accentFillGradientRest,
```

Comment:
Do we want to actually implement this and export with this package? My assumption is no as it would ship to those leveraging the packages. If the intent here was to validate, can we revert this change?

Reply:
Yeah, this was mostly for validation. It could definitely go into another package more specifically for the site or FAST Frame.